openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResource.java

```diff
@@ -164,8 +162,7 @@ public ResultList<TestCase> list(
     ResourceContextInterface resourceContext;
     if (entityLink != null) {
       EntityLink entityLinkParsed = EntityLink.parse(entityLink);
-      filter.addQueryParam(
-          "entityFQN", URLEncoder.encode(entityLinkParsed.getFullyQualifiedFieldValue(), StandardCharsets.UTF_8));
+      filter.addQueryParam("entityFQN", entityLinkParsed.getFullyQualifiedFieldValue());
       resourceContext = TestCaseResourceContext.builder().entityLink(entityLinkParsed).build();
     } else {
```

Comment:
@TeddyCr, here filter param is being encoded. Given this is not stored in the DB and the tests are passing we should be good to go. Please confirm.

Reply:
@sureshms that is my bad I should probably have added a new test, sorry about that. Basically the use case was for tables where column names have spaces in them. entityLink was previously saved as `"entityLink":"<#E::table::rds.dev.dbt_jaffle.table_col_w_space::columns::col%20space>"` (seems like in 0.13.1 this was removed as the entityLink is not encoded which raises an error @ShaileshParmar11). This results in the entityLink being saved encoded in the DB. When the GET request gets processed by the backend it get decoded and we have a miss match between entityLink in the db and entityLink requested.

Reply:
To add on to my previous comment, I tested the behavior with the latest [PR](https://github.com/open-metadata/OpenMetadata/pull/9668) against this file it seems to not break anything. We'll just need to adjust the supported pattern for the entityLink from `"^<#E::\\S+::\\S+>$"` to `"^<#E::\\S+::[\\S\\s]+>$"`.

https://github.com/open-metadata/OpenMetadata/blob/1ec324e43ed4eceb6a7505961d23d019c0cd5091/openmetadata-spec/src/main/resources/json/schema/type/basic.json#L88-L92

![Screenshot 2023-01-11 at 2 52 10 PM](https://user-images.githubusercontent.com/13626425/211823469-af1ba940-ef39-470b-8c46-49304f127d35.png)

Reply:
Thanks @sureshms, makes sense. That should be good.

Reply:
👍